src/App.test.js

```diff
@@ -1,7 +1,7 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render, screen } from "@testing-library/react";
```

Comment:
We can probably remove this file for now

Reply:
Hey @alton-michaux, thanks for your feedback! The test file has been removed.